src/qibolab/platforms/platform.py

```diff
@@ -30,14 +30,37 @@ def disconnect(self):
         self.design.disconnect()
         self.is_connected = False
 
-    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None):
+    def execute_pulse_sequence(
+        self, sequence, nshots=1024, relaxation_time=None, fast_reset=False, sim_time=10e-6, acquisition_type=None
+    ):
         if relaxation_time is None:
             relaxation_time = self.relaxation_time
-        return self.design.play(self.qubits, sequence, nshots=nshots, relaxation_time=relaxation_time)
+        if fast_reset is True:
+            fast_reset = {}
+            for qubit in self.qubits.values():
+                if "c" in str(qubit.name):
+                    pass
+                else:
+                    fast_reset[qubit.name] = self.create_RX_pulse(qubit=qubit.name, start=0)
+        return self.design.play(
+            self.qubits,
+            sequence,
+            nshots=nshots,
+            relaxation_time=relaxation_time,
+            fast_reset=fast_reset,
+            sim_time=sim_time,
+            acquisition_type=acquisition_type,
+        )
 
-    def sweep(self, sequence, *sweepers, nshots=1024, relaxation_time=None, average=True):
+    def sweep(self, sequence, *sweepers, nshots=1024, relaxation_time=None, average=True, sim_time=2e-6):
```

Comment:
Same as above (i.e. document), and I start seeing a pattern in these parameters...
(in the sense that they are all the same)

The separate structure would allow documenting only once, and pass it over and over.